.github/workflows/packages.yml

```diff
@@ -196,18 +198,27 @@ jobs:
           fi
         done
 
-        if [ -n "$packages" ]; then
-          if grep -qP "(^|\\s)${packages// /($|\\s)|(^|\\s)}($|\\s)" ./scripts/big-pkgs.list; then
-            ./scripts/setup-ubuntu.sh
-            sudo apt install ninja-build
-            sudo apt purge -yq $(dpkg -l | grep '^ii' | awk '{ print $2 }' | grep -P '(aspnetcore|cabal-|dotnet-|ghc-|libmono|mongodb-|mysql-|php)') \
-              firefox google-chrome-stable google-cloud-cli microsoft-edge-stable mono-devel mono-runtime-common monodoc-manual powershell ruby
-            sudo apt autoremove -yq
-            sudo apt clean
-            sudo rm -fr /opt/ghc /opt/hostedtoolcache /usr/lib/node_modules /usr/local/share/boost /usr/share/dotnet /usr/share/swift
-          fi
+        if [ ${{ steps.build-info.outputs.docker-build == "false" }} ]; then
+          ./scripts/setup-ubuntu.sh
+          sudo apt install ninja-build
```

Comment:
termux_setup_ninja not good?

Reply:
This is much simpler, though I suppose more likely to get out of date?